README.md

```diff
@@ -28,6 +41,48 @@
   array.map(iteratee)
   ```
 
+#### ReduceRight
+
+*  Underscore
+
+  ```javascript
+  _.reduceRight(array, iteratee)
+  ```
+
+*  ES5.1
+
+  ```javascript
+  array.reduceRight(iteratee)
+  ```
+
+#### Every
```

Comment:
The heading should be a short description of what the example does, e.g. "Test if all elements in the array pass the predicate"